<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configurator</class>
 <widget class="QMainWindow" name="Configurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EyeBrowse - Configurator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Automatic LockScreen</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Delay before lock</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="updDelayLock">
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>seconds</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="ckbLockScreen">
         <property name="text">
          <string>Activated</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Automatic ScrollDown</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Number of ticks by scroll</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cooldown between scrolls</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="ckbScrollDown">
         <property name="text">
          <string>Activated</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>seconds</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="updScrollDownDelay">
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ticks</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="updScrollTicks">
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>80</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Configurator</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>Configurator</receiver>
   <slot>s_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_save()</slot>
 </slots>
</ui>
